### DISCUSSION TITLE
pen starts selecting region

### DISCUSSION THREAD

**Discussion by p0gster** [xournalpp/xournalpp#4863] [Q&A] [answered]
When I draw with the pen tool, it appears to select a region as well as draw. The region select tool is not selected and the region remains until it is removed with an eraser.  This is shown here.
![image](https://user-images.githubusercontent.com/132705326/236548186-b7eba0e8-3cf5-4cc4-a9b6-d8e7b6f520ef.png)
How do i stop this behaviour?

**Comment by rolandlo** [comment, marked as answer]
Toggle the Fill option. You can find it in the toolbar next to the colors. You can also find it under menu `Tools > Pen Options > Fill`.
![grafik](https://user-images.githubusercontent.com/40485433/236549174-b38d1daf-112d-4239-97e6-746c38f1bea3.png)

**Reply by p0gster** [reply]
thanks that helped

**Comment by rolandlo** [comment]
You should also toggle the line style to solid.

**Reply by p0gster** [reply]
done
